libs/aggregation_example/src/aggregation.rs

```diff
@@ -0,0 +1,29 @@
+//! Example of an aggregation struct to be used in storage
+#[derive(Clone, Debug)]
+pub struct AggregationStruct {
```

Comment:
This is less aggregation and more average?

Reply:
Yes, I'm making them all called AggregationStruct because they all aggregate, and the simulator needs to know the name of the struct.

Reply:
But the function name is aggregation.rs. This seems incorrect?

Reply:
Oh, I see.  It should be avg.rs now.